bench/Makefile

```
@@ -0,0 +1,50 @@
+CORE_TARGETS = bench_init nested_retain warm_holder abi_check
+OPTIONAL_TARGETS = phase_probe
+
+HAVE_NVML := $(shell ldconfig -p | grep -q libnvidia-ml && echo 1)
+
+TARGETS = $(CORE_TARGETS)
+ifeq ($(HAVE_NVML),1)
+TARGETS += $(OPTIONAL_TARGETS)
+endif
+
+.PHONY: all clean check-cuda-home print-targets
+
```

checkmake (0.3.2)

[warning] Required target "test" is missing from the Makefile.

(minphony)